its/src/test/java/com/sonar/it/scanner/msbuild/sonarqube/SslTest.java

```diff
@@ -104,21 +100,23 @@ static void init() {
   @Test
   void trustedSelfSignedCertificate() throws Exception {
     try (var server = initSslTestAndServer(keystorePath, keystorePassword)) {
+      var projectKey = "trustedSelfSignedCertificate";
```

Comment:
We should use `TestName` rule. https://www.baeldung.com/junit-get-name-of-currently-executing-test#1-individual-test-1

Reply:
https://sonarsource.atlassian.net/browse/SCAN4NET-298